pkg/git/git.go

```diff
@@ -74,6 +76,10 @@ func Fetch(logger *zap.SugaredLogger, spec FetchSpec) error {
 	if _, err := run(logger, "", "remote", "add", "origin", trimmedURL); err != nil {
 		return err
 	}
+	if _, err := run(logger, "", "config", "--global", "http.sslVerify", strconv.FormatBool(spec.SSLVerify)); err != nil {
+		logger.Warnf("Failed to set http.sslVerify in git config: %s", err)
+		return err
+	}
```

Comment:
it looks like this module has no tests yet :fearful: but if you felt super motivated and wanted to add a unit test for this to get it started that would be amazing :pray: (no worries if not tho!)

Reply:
Sadly all out of super motivation due to pending vacation and deadlines ....... standard motivation remains so I did add an extra test elsewhere.